assets/reviews.js

```diff
@@ -9,32 +9,36 @@
 
   try {
     const reviews = await youcanjs.product.fetchReviews(productId).data();
+    const maxReviewsToShow = 3;
 
     reviewsContainer.style.display = 'block';
-    reviews.forEach((review) => {
+
+    for (let i = 0; i < maxReviewsToShow && i < reviews.length; i++) {
+      const review = reviews[i];
       const reviewItem = document.createElement('li');
+
       reviewItem.classList.add('review-item');
-      reviewItem.innerHTML = `
-        <div class='header'>
-          <div class="profil">
-            <img loading='lazy' class='image' src=${review.images_urls[0] || defaultAvatar} />
-            <div class='info'>
-              <span class='name'>${review.first_name || ''} ${review.last_name || ''}</span>
-              <span class='created-at-date'>${review.created_at}</span>
-            </div>
-          </div>
-          <div class='yc-reviews-stars'
-              style="--rating: ${review.ratings};"
-              aria-label="Rating of this product is ${review.ratings} out of 5"
-          >
-          </div>
-        </div>
-        <div class='content'>
-          ${review.content === null ? '' : review.content}
-        </div>
-      `;
+      createElement(reviewItem, review);
       reviewsWrapper.appendChild(reviewItem);
-    });
+    }
+
+    if(reviews.length > maxReviewsToShow) {
+      const showMoreButton = $('#show-more');
+
+      showMoreButton.style.display = "block";
+      showMoreButton?.addEventListener("click", () => {
+        for (let i = maxReviewsToShow; i < reviews.length; i++) {
+          const review = reviews[i];
+          const reviewItem = document.createElement('li');
+
+          reviewItem.classList.add('review-item');
+          createElement(reviewItem, review);
+          reviewsWrapper.appendChild(reviewItem);
+        }
+
+        showMoreButton.style.display = "none";
+      });
+    }
```

Comment:
the reviews pagination show more button should'n be just display: none.
you should add it using the sdk , we already have the pagination on the reviews.
<img width="606" alt="Screen Shot 2023-03-08 at 09 35 47" src="https://user-images.githubusercontent.com/108662497/223663517-a554abba-e0b9-4dbb-ae9e-c13c140a597e.png">

Reply:
> Missing the multi column block <img alt="Screen Shot 2023-03-08 at 09 31 16" width="428" src="https://user-images.githubusercontent.com/108662497/223662602-df4029a6-769c-45b4-8d54-4de6458cf4e4.png">

it isn't included in the ticket and also we don't know how the implementation of it should be like?

Reply:
> the reviews pagination show more button should'n be just display: none. you should add it using the sdk , we already have the pagination on the reviews. <img alt="Screen Shot 2023-03-08 at 09 35 47" width="606" src="https://user-images.githubusercontent.com/108662497/223663517-a554abba-e0b9-4dbb-ae9e-c13c140a597e.png">

it just a button it's one element it won't affect the performance of the web page

Reply:
you are skipped the section of multi column we already have it on the home page just include it on the product page also
<img width="250" alt="Screen Shot 2023-03-08 at 09 43 39" src="https://user-images.githubusercontent.com/108662497/223665324-78d7ddd2-91ae-4505-927e-92bd91d56b2e.png">

Reply:
what is the purpose of adding a show more button if we just add display none display block ?

Reply:
display none and block it is a behaviour effect on the button only and for the reviews it's added from the backend

Reply:
For the reviews, I would say that the proper way is when the user clicks on load more button to load more views, it should call the API using the SDK

You should specify to the SDK how many reviews to return each call, check this (check this)[https://github.com/youcan-shop/youcan-js#pagination]

For the multi column block, not sure tbh, you have two choices
User will add an image in the product description in the seller area which he prefers 
Create a block that can help him doing that, like you did with the countdown